src/Tools/ExternalAccess/FSharp/TaskList/FSharpTaskListDescriptor.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.TaskList;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.TaskList
+{
+    internal readonly struct FSharpTaskListDescriptor
+    {
+        internal readonly TaskListItemDescriptor Descriptor;
```

Comment:
Private?

Reply:
Is needed on the way back (when data is returned from F#) to retrieve the original priority.
The F# impl does not need to know about priority, it will just flow together with TaskListItemDescriptor  using this struct.

Reply:
You can just have the priority be internal here.  I don't think you should have teh descriptor as it woudl then expose the descriptor to F#, which is what we're trying to expose.